mindsdb/utilities/config.py

```diff
@@ -495,7 +495,7 @@ def raise_warnings(self, logger) -> None:
 
         for env_name in ('MINDSDB_HTTP_SERVER_TYPE', 'MINDSDB_DEFAULT_SERVER'):
             env_value = os.environ.get(env_name, '')
-            if env_value.lower() not in ('waitress', 'flask', 'gunicorn'):
+            if env_value.lower() not in ('waitress', 'flask', 'gunicorn', ''):
```

Comment:
does adding of '' change the behavior?

Reply:
Isn't waitress the default? If so, we need to remove the warning bellow since it is confusing

Reply:
added '' deletes the warning